docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,12 +41,111 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### Decision Table
```

Comment:
But currently, it's in enrichers in the creator panel.
![Screenshot 2024-05-24 at 13 16 19](https://github.com/TouK/nussknacker/assets/158037696/e56a32e3-106b-4d35-849c-e4ebbe1daa1d)

The Product Team agrees that it's not an enricher and it will be changed in the future, but for now, let's have it here and be consistent with the Nu Designer UI.